plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/EnrichMediatorPropertiesEditionPartForm.java

```diff
@@ -1553,42 +1549,48 @@ protected Composite createtargetXPathWidget(FormToolkit widgetFactory, final Com
         targetXPathText.setLayoutData(valueData);
-        targetXPathText.addFocusListener(new FocusAdapter() {
-            /**
-             * @see org.eclipse.swt.events.FocusAdapter#focusLost(org.eclipse.swt.events.FocusEvent)
-             * 
-             */
+        
+        targetXPathText.addMouseListener(new MouseAdapter() {
+            
             @Override
-            @SuppressWarnings("synthetic-access")
-            public void focusLost(FocusEvent e) {
+            public void mouseDown( MouseEvent event ) {
+                openNamespacedPropertyEditor(parent);
             }
-
-            /**
-             * @see org.eclipse.swt.events.FocusAdapter#focusGained(org.eclipse.swt.events.FocusEvent)
-             */
+            
+        });
+        
+        targetXPathText.addKeyListener(new KeyListener() {
+                        
             @Override
-            public void focusGained(FocusEvent e) {
-                EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
-                        SWT.NULL, targetXPath);
-                // valueExpression.setPropertyValue(valueExpressionText.getText());
-                nspd.open();
-                targetXPathText.setText(targetXPath.getPropertyValue());
-                propertiesEditionComponent
-                        .firePropertiesChanged(new PropertiesEditionEvent(EnrichMediatorPropertiesEditionPartForm.this,
-                                EsbViewsRepository.EnrichMediator.Target.targetXPath, PropertiesEditionEvent.COMMIT,
-                                PropertiesEditionEvent.SET, null, getTargetXPath()));
+            public void keyPressed(KeyEvent e) {
+                openNamespacedPropertyEditor(parent);
             }
+            
+            @Override
+            public void keyReleased(KeyEvent e) {}
+            
         });
+        
         EditingUtils.setID(targetXPathText, EsbViewsRepository.EnrichMediator.Target.targetXPath);
         EditingUtils.setEEFtype(targetXPathText, "eef::Text");
         Control targetXPathHelp = FormUtils.createHelpButton(widgetFactory, parent, propertiesEditionComponent
                 .getHelpContent(EsbViewsRepository.EnrichMediator.Target.targetXPath, EsbViewsRepository.FORM_KIND),
                 null); // $NON-NLS-1$
-        // aggregationExpressionElements = new Control[] { aggregationExpressionLabel, aggregationExpressionText,
-        // aggregationExpressionHelp };
         targetXPathElements = new Control[] { targetXPathLabel, targetXPathText, targetXPathHelp };
         return parent;
     }
+    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, targetXPath);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```

Comment:
Remove commented code.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/8bf90d13170529d3776b1699ed2b01a4b8b110d9